moment.d.ts

```diff
@@ -716,6 +716,19 @@ declare namespace moment {
 
   export var defaultFormat: string;
   export var defaultFormatUtc: string;
+  
+  export var HTML5_FMT: { 
+    DATETIME_LOCAL:string,
```

Comment:
I would guess that the styling here would need to be updated prior to it being approved for merging so that like the code above it, there should be a space after the colon and the type.

Reply:
Oh good call. I have updated it.